### PR TITLE
[sdk-core][sdk-redux] add GDA pool subgraph entity queries

### PR DESCRIPTION
**Pull request by kasparkallas**
### Why?
The queries are already used in the Console. Moving these queries upstream. Will re-use them in the Dashboard.

Includes other small fixes and updated subgraph schema.

### CODE REVIEW

**Comment by hellwolf** [comment]
Pending subgraph fix.

File: packages/subgraph/src/mappings/superfluidPool.ts [hunk 1]
```diff
@@ -61,6 +61,11 @@ export function handleMemberUnitsUpdated(event: MemberUnitsUpdated): void {
     pool = updatePoolTotalAmountFlowedAndDistributed(event, pool);
     settlePDPoolMemberMU(pool, poolMember, event.block);
 
+<<<<<<< HEAD
```
Comment (hellwolf) [inline packages/subgraph/src/mappings/superfluidPool.ts]: merge conflit

File: packages/subgraph/config/hardhat.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"network":"mainnet","testNetwork":"hardhat","hostStartBlock":0,"hostAddress":"0x4374EEcaAD0Dcaa149CfFc160d5a0552B1D092b0","cfaAddress":"0x44BF2a9217A2970A1bCC7529Bf1d40828C594320","idaAddress":"0x0826223156fF61F9abf620D2f58A06177DA664Cc","gdaAddress":"0x1Ad83f2789e013a41eA18819F0e24a4d41477b4c","superTokenFactoryAddress":"0x9b9389FacF9d345676b178EfEe328334cA711A38","resolverV1Address":"0x0e528E13E32cfB153818F1e10c7aff11ec5B46A3","nativeAssetSuperTokenAddress":"0x68bFD28e2AB62C450C563E1687f5f4F6b008149f","constantOutflowNFTAddress":"0x765FEFc8D7b2A6ec1D83626c50781e8cdd7e8284","constantInflowNFTAddress":"0x0983210c9f6f968ACC2010ac4dc56124dcDe2EC2"}
```
Comment (hellwolf) [inline packages/subgraph/config/hardhat.json]: was it meant to be added?

Comment (0xdavinchee) [reply packages/subgraph/config/hardhat.json]: ditto w/ `polygon-mainnet.json`

Comment (kasparkallas) [reply packages/subgraph/config/hardhat.json]: removed, added to gitignore

only `mock.json` is commited in that folder

File: packages/subgraph/src/mappings/superfluidPool.ts (hunk 1 above)
Comment (kasparkallas) [reply packages/subgraph/src/mappings/superfluidPool.ts]: fixed, thank you!

**Review by 0xdavinchee — APPROVED** [review]
LGTM

**Pull request merged**